js/script.js

```diff
@@ -0,0 +1,72 @@
+// console.log("working!!!!");
+
+const APIKEY = "82f2229c255b1505c496b13203e48e52";
+const BASE = "https://api.openweathermap.org/data/2.5/forecast";
+const CITYID = "?id=2964574";
+const UNITS = "&units=metric";
+const URL = `${BASE}${CITYID}${UNITS}&APPID=${APIKEY}`;
+
+function getDay(longDate) {
+  let date = new Date(longDate);
+  let day = date.toString().substring(0, 3);
+  return day;
+}
+
+fetch(URL).then(response => {
+  response.json().then(res => {
+    //header
+    document.getElementsByTagName("header")[0].innerHTML = `Weather in
+    ${res.city.name}, ${res.city.country}`;
+    //day
+    let forecast = res.list;
+
+    //temp
+    document.getElementById("today-temp").innerHTML = `${Math.round(
+      forecast[0].main.temp
+    )}&#176;`;
+
+    let icon = `<i class="wi wi-${forecast[0].weather[0].icon}"></i>`;
+    document.getElementById("today-icon").innerHTML = icon;
+    document.getElementById("today-description").innerHTML =
+      forecast[0].weather[0].description;
+    document.getElementById(
+      "today-minMax"
+    ).innerHTML = `<div class="details today-details">Max: ${Math.round(
+      forecast[0].main.temp_max
+    )}&#176; | Min: ${Math.round(forecast[0].main.temp_min)}&#176;</div>`;
+
+    // next days forecast
+
+    let data = [];
+    for (let i = 8; i < forecast.length; i += 8) {
+      data.push(forecast[i]);
+    }
+    data.map((item, index) => {
```

Comment:
Minor thing but you don't need to use `map` here, you can use `forEach` (which loops through the array in the same way, but also returns a new array). Generally you use map when you want a different version of the same array.

For example if  wanted to do it in a way that's that's a bit more functional, and had to use map, you could do.
    
    sections = data.map( (item, index) => {
        //do everything to create the section element
        return section;
    })
    sections.forEach(section => aside.append(section));

It doesn't make any difference, but it is more clear why you're using one or the other that way.

Reply:
So really, should I use forEach whenever possible? I tend to use map instead, I think I just prefer it to be honest — would that be considered incorrect then? Thanks.